src/NSwag.CodeGeneration.CSharp/Templates/Client.Class.liquid

```diff
@@ -279,7 +292,11 @@
                             headers_[item_.Key] = item_.Value;
                     }
 
+{% if GeneratePrepareRequestAndProcessResponseAsAsyncMethods %}
```

Comment:
Change `%}` to `-%}` in all ifs so that the line is removed in the output...

Reply:
Done!